src/components.ts

```diff
@@ -149,6 +150,11 @@ export async function initComponents(
 
   const alchemyNftFetcher = await createAlchemyNftFetcher({ config, logs, fetch })
 
+  const nameOwnerFetcher = createElementsFetcherComponent({ logs }, async (name) => {
+    const result = await fetchNameOwner({ theGraph }, name)
+    return [result]
```

Comment:
If you're re-using the `createElementsFetcherComponent` and the `result` is undefined, what do you think about returning an empty array?